networks.yaml

```diff
@@ -19,15 +19,23 @@ matic:
       address: '0x8B9805A2f595B6705e74F7310829f2d299D21522'
       transactionHash: '0x1af1b88e36bb0289f6e037e00d01b26bed997ce7a0406ebafce07dcf766ce7c6'
       startBlock: 4023693
+    ERC20Detailed:
+      address: "0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174"
+      transactionHash: "0x51103659d8827d6d24732d1168de73390f4f387d58a8281944241657f1bc61d7"
+      startBlock: 5013591
   networkName: matic
 mumbai:
   contracts:
     ConditionalTokens:
       address: '0x7D8610E9567d2a6C9FBf66a5A13E9Ba8bb120d43'
       transactionHash: '0xb3ce795972a85754f8803d6b43408fc71f472733766236072a270b04ffbb937b'
-      startBlock: 3456411
+      startBlock: 26000000
```

Comment:
@cristiangpr Same question here

Reply:
It's the deployment block of the USDC contract